types/macaca-circular-json/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for circular-json 0.4
```

Comment:
Why is the package `circular-json` but the folder name is `macaca-circular-json`

Reply:
> Why is the package `circular-json` but the folder name is `macaca-circular-json`

fixed, thx~